demos/python_demos/llm_text_generation/README.md

```diff
@@ -0,0 +1,79 @@
+# Llama demo with python node {#ovms_demo_python_llama}
+
+## Build image
+
+From the root of the repository run:
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+make python_image
+```
+
+## Download model
+
+We are going to use [red-pajama-3b-chat](https://huggingface.co/togethercomputer/RedPajama-INCITE-Chat-3B-v1) model in this scenario.
+Download the model using `download_model.py` script:
+
+```bash
+cd demos/python_demos/llm_text_generation
+pip install -r requirements.txt
+python3 download_model.py
+```
+
+The model will appear in `./model` directory.
+
+## Deploy OpenVINO Model Server with the Python Calculator
+
+Mount the `./model` directory with the model.  
+Mount the `./servable_unary_version` or `./servable_stream_version` which contains:
+- `model.py` and `config.py` - python scripts which are required for execution and use [Hugging Face](https://huggingface.co/) utilities with [optimum-intel](https://github.com/huggingface/optimum-intel) acceleration.
+- `config.json` - which defines which servables should be loaded
+- `graph.pbtxt` - which defines MediaPipe graph containing python calculator
+
+Depending on the use case, `./servable_unary_version` and `./servable_stream_version` showcase different approach:
+- *unary* - single request - single response, useful for low latency calls with no intermediate results
```

Comment:
"low latency calls" might be confusing, someone can think that they should use it for low latency (better performance)